### PR TITLE
Javascript/javascript1/week1

### PR DESCRIPTION
**Pull request by shanhaider23**
JavaScript week 01 Homework

### CODE REVIEW

File: javascript/javascript1/week1/startup_name.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+let firstWords =['Flex', 'Perfect', 'Extreme', 'Fort', 'Newly']
+let secondWords = ['Corporate', 'IPS', 'APS', 'Limited', 'Organization']
+let randomNumber = Math.floor(Math.random(0) * 10) + 0;
```
Comment (flom84) [inline javascript/javascript1/week1/startup_name.js]:
```suggestion
let randomNumber = Math.random() *10
```
What do you think?

File: javascript/javascript1/week1/dog_age.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+let dogYearOfBirth = 2018
+let dogYearFuture = 2027
+let dogYear =dogYearFuture - dogYearOfBirth
+let dogAgeInHumanYear = 16*Math.log(dogYear)+31;
```
Comment (flom84) [inline javascript/javascript1/week1/dog_age.js]: Just a simple multiplication is needed here. `dogAge=7*humanAge`

Comment (shanhaider23) [reply javascript/javascript1/week1/dog_age.js]: Thanks Yes you are right I am going to replace it.

File: javascript/javascript1/week1/startup_name.js (hunk 1 above)
Comment (shanhaider23) [reply javascript/javascript1/week1/startup_name.js]: I have tried this code but I don't know my startUpName variable become undefined

Comment (flom84) [reply javascript/javascript1/week1/startup_name.js]: Yes. I have forgot to add Math.floor
```suggestion
let randomNumber =   Math.floor(Math.random() *firstWords.length) 
```

Comment (shanhaider23) [reply javascript/javascript1/week1/startup_name.js]: Thanks :) I will change it.

**Pull request merged**